.docker/cypress/spec/cypress/support/commands.js

```diff
@@ -65,6 +65,42 @@ Cypress.Commands.add("getCSRFToken", () => {
   })
 })
 
+Cypress.Commands.add('getInputByText', { prevSubject: 'optional'}, (subject, text, checkbox) => {
+  if (subject) {
+    if (checkbox)
+      return cy.wrap(subject, { log: false }).contains(text).closest('.control-group', { log: false })
+        .contains(checkbox).find('input[type="checkbox"]', { log: false }).first({ log: false })
+      return cy.wrap(subject, { log: false }).contains(text).closest('.control-group', { log: false })
+        .find('input:not([type="hidden"]), textarea, select', { log: false }).first({ log: false })
```

Comment:
Ich bin grundsätzlich fan von kurzen Schreibweisen, allerdings leidet in meinen Augen hier die Lesbarkeit. Ich versuche grundsätzlich if/else immer mit geschweiften Klammern zu schreiben. Als Ausnahme dazu sehe ich kurze Statements, welche dann den Scope verlassen (z.B. `return false`). Hier (und auch weiter unten) finde ich aber die beiden returns sehr verwirrend…